azure-pipelines.yml

```diff
@@ -2,13 +2,13 @@ pr:
 - 'main'
 
 pool:
-  vmImage: 'windows-latest'
+  vmImage: 'windows-2022'
 
 steps:
 - task: BatchScript@1
   displayName: 'Run VsDevCmd.bat'
   inputs:
-    filename: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat'
+    filename: 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat'
```

Comment:
>2022 [](http://example.com/codeflow?start=56&length=4)

we could also update re2.native.sln to use the latest bits.

f you open the re2.native.sln, check the properties of each project and you will see that it is using VS2019 + a specific version.

Reply:
Tried, looks like this change does not currently work. Discussed with Eddy offline.